client/src/luigi-client.js

```diff
@@ -71,13 +67,13 @@ function luigiClientInit() {
       }
     }
     currentContext = rawData;
-  }
+  };
 
-  function setAuthData(eventPayload) {
+  const setAuthData = eventPayload => {
     if (eventPayload) {
       authData = eventPayload;
     }
-  }
+  };
 
   window.addEventListener('message', function messageListener(e) {
```

Comment:
As discussed, let´s have a unique event listener for `message` event ;)

Reply:
Will be done in separate PR for this ticket, for easier review.

Reply:
PR: https://github.com/SAP/luigi/pull/559